main/network.c

```diff
@@ -315,16 +315,23 @@ static inline void sub_times(struct timeval a, struct timeval b, struct timeval
 	}
 }
 
-static inline void php_network_set_limit_time(struct timeval *limit_time,
+static inline zend_result php_network_set_limit_time(struct timeval *limit_time,
 		struct timeval *timeout)
 {
 	gettimeofday(limit_time, NULL);
+	const double timeoutmax = (double) PHP_TIMEOUT_ULL_MAX / 1000000.0;
```

Comment:
I'm just checking the whole logic here and I wonder why PHP_TIMEOUT_ULL_MAX is unsigned int 64? The time_t is signed int 64 (or 32 on 32bit arch) if I'm not mistakend. And in this case this is later converted to int (see php_pollfd_for and mainly php_tvtoto). So what am I missing?